socorro/unittest/external/boto/test_upload_telemetry_schema.py

```diff
@@ -7,7 +7,11 @@
 
 
 class TestUploadTelemetrySchema:
-    def test_upload_worked(self, boto_helper, caplogpp):
+    def test_upload_worked(self, boto_helper, caplogpp, monkeypatch):
+        # NOTE(willkg): configman apps look at sys.argv for command line arguments, so
+        # we monkey patch sys.argv so it doesn't pick up pytest arguments and errors
+        # out
+        monkeypatch.setattr("sys.argv", ["upload_telemetry_schema.py"])
```

Comment:
This bug has been incredibly annoying for the last week or two. I finally got around to fixing it today.